runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -157,6 +157,14 @@ template <typename LoaderDomain> class BaseLoader
   void loadSpaceToDepth(const Operator *op, ir::Graph &subg);
   void loadLeakyRelu(const Operator *op, ir::Graph &subg);
 
+  void verifySubgraphIndex(int subg_index)
```

Comment:
Looks better to check on `OperationValidator`, IMO.

Reply:
`OperationValidator` cannot access other subgraphs. Anyway I also think that it would be better to verify this in core of `onert`.

Reply:
@hseok-oh Maybe we should distinguish which is checked from model-loading or from compilation.

Reply:
If we need to check on model-loading, we can move calling `OperationValidator` from `Compiler::compiler()` to `Graph::finishBuilding()`